components/Datepicker/react.stories.js

```diff
@@ -0,0 +1,25 @@
+import { storiesOf } from '@storybook/react'
+import React, { useState } from 'react'
+// import classNames from 'classnames/bind'
```

Comment:
These commented lines can be removed. Also, the entire stories.module.scss file can be deleted.